debian/usr.bin.snap-confine

```diff
@@ -1,7 +1,7 @@
 # Author: Jamie Strandboge <jamie@canonical.com>
```

Comment:
Ack

(this will require some changes to packaging that is somewhat harder to coordinate)